readme.md

```diff
@@ -178,6 +178,12 @@ Type: `Object` `Array` `number` `string` `boolean` `null`
 
 JSON body. The `Content-Type` header will be set to `application/json` if it's not defined.
 
+###### userData
+
+Type: `any`
```

Comment:
For type safety, would this not be better as `unknown`?

Reply:
Indeed. Also I forgot to add it to the types.